parse/TechsParser.cpp

```diff
@@ -116,14 +116,14 @@ namespace {
 
             prerequisites
                 =   parse::label(Prerequisites_token)
-                >  (    '[' > +tok.string [ insert(_r1, _1) ] > ']'
+                >  (    ('[' > +tok.string [ insert(_r1, _1) ] > ']')
```

Comment:
Ok, I'll try.

I do not think that adding brackets to the parser code increases its clarity.  That is why I put all the parser stuff in a separate commit.  

I almost mentioned it specifically in the PR because the `>` operator is not a comparison operator so the warning itself is misleading.